app/views/rosters/index.haml

```diff
@@ -6,6 +6,7 @@
       %th Name
       %th Fallback
       %th Switchover
+      %th Number
```

Comment:
Any reason for 'Number' rather than 'Phone'?

Reply:
Hmmm... nope. As I was writing this feature, I kept mixing up in my head which one I wanted to use in general. But they _should_ be consistent.